index.html

```diff
@@ -118,6 +119,13 @@ <h6>Industry Sectors</h6>
                           <li class="nav__item">
                             <a class="nav__item-link" href="industries-single-sector.html">Oil & Gas</a>
                           </li><!-- /.nav-item -->
+                          <li class="nav__item">
+                         <a class="nav__item-link" href="Product.html"> Rotary Capper </a>

```

Comment:
Is it right place ?
We have asked you to ensure there is category and underneath there should be product.

Industry Sectors is not category.
You should have product category as per issue we mentioned - Capping Machine.